src/wasm-lib/kcl/src/coredump/mod.rs

```diff
@@ -74,18 +85,44 @@ pub trait CoreDump: Clone {
             webrtc_stats,
             github_issue_url: None,
             pool: self.pool()?,
+            client_state,
         };
-        app_info.set_github_issue_url(&screenshot_url)?;
 
-        Ok(app_info)
+        // pretty-printed JSON byte vector of the coredump.
+        let data = serde_json::to_vec_pretty(&core_dump_info)?;
+
+        // Upload the coredump.
+        let links = zoo_client
+            .meta()
+            .create_debug_uploads(vec![kittycad::types::multipart::Attachment {
+                name: "".to_string(),
+                filename: Some(format!(r#"modeling-app/coredump-{}.json"#, coredump_id)),
+                content_type: Some("application/json".to_string()),
+                data,
+            }])
+            .await
+            .map_err(|e| anyhow::anyhow!(e.to_string()))?;
```

Comment:
Here `e` is already `anyhow::Error` so this is doing a useless conversion to the same type
```suggestion
?;
```

Reply:
@adamchalmers Tried this one, but it threw:

> error[E0277]: `*mut u8` cannot be shared between threads safely

🙅 Reverted